libraries/rush-lib/src/cli/test/RushCommandLineParser.test.ts

```diff
@@ -104,7 +105,10 @@ async function getCommandLineParserInstanceAsync(
   // Bulk tasks are hard-coded to expect install to have been completed. So, ensure the last-link.flag
   // file exists and is valid
   // TODO: Support subspaces
```

Comment:
Is this true now?

Reply:
+1 for this question.

The "last-link.flag" seems not related with the subspace feature and it is planned to be deprecated. I suppose it's not true.

Reply:
Removed TODO